tests/benchdnn/rnn/ref_rnn_bwd.cpp

```diff
@@ -240,7 +233,7 @@ void rnn_linear_bwd(const prb_t &prb, const args_t &args,
     AOC<float> diff_bias(diff_bias_, prb.n_layer, prb.n_dir(),
             prb.n_gates() + is_lbr, prb.dhc);
 
-    std::vector<float> ws_bwd_buffer;
+    float *ws_bwd_buffer = (float *)zmalloc(prb.get_ws_size(FLAG_BWD), 64);
```

Comment:
Use `std::unique_ptr` to avoid the manual memory management.

Reply:
Discussed offline: it will require extra manipulations such as custom deleter through zfree, which unlikely make the code simpler and more readable.